stable/jenkins/values.yaml

```diff
@@ -248,6 +248,7 @@ Master:
   # the jenkins config with the contents of the configMap every time the pod starts.
   OverwriteConfig: false
 
+  UseIngress: false
```

Comment:
Sure, can be, I was just scanning other boolean usages in the same chart, but I see that "Enabled" within the immediate context is used too. I'll make the change...the only weird thing I thought was that HostName was left outside the ingress context but controlled it before which led me down that same path?

Reply:
I completely understand why you chose this path.
Nevertheless, this chart is currently not following the [chart best practices](https://helm.sh/docs/chart_best_practices).

If you are willing to extend your contribution and make the entire `ingress` section more compliant with the above mentioned guidelines, it will be highly appreciated!

You can look at the same section in the [grafana](https://github.com/helm/charts/blob/master/stable/grafana/values.yaml#L82) chart for reference ;)

Reply:
Already updated, please check